projects/rhai/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder-rust
+# TODO: Change this back to the main repository
+#RUN git clone --depth 1 https://github.com/rhaiscript/rhai.git rhai
+RUN git clone --depth 1 https://github.com/silvergasp/rhai.git rhai
```

Comment:
> And how to change back to main fork?

I've currently got the integration pointed at my personal branch, so that I can test the integration. I'll change it back to the main repo once https://github.com/rhaiscript/rhai/pull/774 is merged.